lib/engine/game/g_18_norway/corporations.rb

```diff
@@ -96,6 +96,19 @@ module Corporations
             coordinates: 'G31',
             color: :'#95c054',
           },
+          {
+            floatable: false,
+            hide_shares: true,
+            sym: 'NSB',
+            name: 'Norges Statsbaner',
+            tokens: [],
+            shares: [100],
```

Comment:
Is there something in place to stop this corporation being floated by a player?

Reply:
Yes there were a stop but that appeared only after you did try to par.
Added can_par? to make it even more visable